x-pack/solutions/security/packages/kbn-securitysolution-endpoint-exceptions-common/api/delete_endpoint_list_item/delete_endpoint_list_item.schema.yaml

```
@@ -9,7 +9,7 @@ paths:
       x-codegen-enabled: true
       operationId: DeleteEndpointListItem
       summary: Delete an endpoint exception list item
-      description: Delete an endpoint exception list item using the `id` or `item_id` field.
+      description: Delete an Elastic Endpoint exception list item by using the `id` or `item_id` field.
```

The "by using" phrasing doesn't seem common, and it reads a little awkwardly to me.

```suggestion
      description: Delete an Elastic Endpoint exception list item specified by the `id` or `item_id` field.
```

---

Oh yeah, this is much better. Fixed in [`9790ae0` (#223053)](https://github.com/elastic/kibana/pull/223053/commits/9790ae0c6358575f1412db13814abd63673de5db).